src/layouts/georestrictedModal.tsx

```diff
@@ -0,0 +1,59 @@
+import { Dispatch, FC, SetStateAction, useMemo } from 'react'
+import Lottie from 'lottie-react'
+import { Dialog, DialogBody, DialogCloseDefault, DialogContent, DialogOverlay } from 'gfx-component-lib'
+import GeoRestrictedDark from '../animations/GeorestrictedDark.json'
+import GeoRestrictedLite from '../animations/GeorestrictedLite.json'
+import { useDarkMode } from '@/context'
+import useBreakPoint from '@/hooks/useBreakPoint'
+
+export const GeorestrictionModal: FC<{
+  geoBlocked: boolean
+  setGeoBlocked: Dispatch<SetStateAction<boolean>>
+}> = ({ geoBlocked, setGeoBlocked }) => {
+  const { mode } = useDarkMode()
+  const breakpoint = useBreakPoint()
+
+  const Content = useMemo(
```

Comment:
It should be in separate component, there is to need to make this component bulky